### ISSUE TITLE
Restart buttons need to have some visual indication of interaction

### ISSUE THREAD

**Issue by alexwykoff** [brave/browser-laptop#7968]
- Did you search for similar issues before submitting this one?
Yes

- Describe the issue you encountered:
When choosing 'Yes' from the restart notification bar, a few seconds would pass between the click and the restart. My first impression was that I didn't click or I missed the target. 

- Platform (Win7, 8, 10? macOS? Linux distro?):
Discovered on OS X

- Brave Version (revision SHA):
0.14.0 RC2 (b00c5d9)

- Steps to reproduce:
    1. Open Preferences
    2. Change your language
    3. Click 'Yes' in the restart notification bar
    4. Observe the interaction
    
- Actual result:
On a busy system, there will be a considerable delay which makes it appear as though no user action occurred.

- Expected result:
Visual feedback should let the user know they hit the target and a restart is imminent.

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
Yes

- Is this an issue in the currently released version?
Not sure

- Can this issue be consistently reproduced?
Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by bsclifton** [comment]
Moving to 0.14.3.  Before we can handle this, we'll need to understand what we'd like the user to experience. cc: @bradleyrichter.  Do we put a modal / overlay over the UI with a spinner?

**Comment by bradleyrichter** [comment]
probably a good case for a modal because heading down a new path while waiting for the restart will only cause more frustration when it closes on you mid-click/read.

**Comment by alexwykoff** [comment]
A setTimeout might work so that we show an animation before the restart action kicks off

**Comment by I3uckwheat** [comment]
I can work on this! Could take me a bit to get familiar with what is going on with the code though!

**Comment by bsclifton** [comment]
Closing as `wontfix` - this experience will change as we move to [brave-core](https://github.com/brave/brave-browser)
